### ISSUE TITLE
Feature Idea: Bookmark or favorites boards

### ISSUE THREAD

**Issue by coltoneshaw** [mattermost/focalboard#1975]
## Summary

Right now I'm testing using a personal Board for my daily tasks and such. What I'm finding is that I have to keep reopening that board multiple times a day either through search or DM > Boards. This adds a few more steps to the process that what is ideal. It would be awesome to be able to open a specific tab on the desktop app for a board, or to bookmark the board 

![Screen Shot 2021-12-13 at 9 58 41 AM](https://user-images.githubusercontent.com/46071821/145835479-aa610401-4691-4412-a798-8032b6966477.png)

## How important this is to me and why

Importance: High

Use cases:
1. This would allow you to use boards and not have to constantly search for the specific channel, then open the board. This gets especially cumbersome if you're working on boards a lot. 
2. Overall reduces number of clicks / interactions to your important content

## Additional context/similar features
Something like "Recent boards" and showing the last 5-10 you viewed would be another huge quality of life improvement.

**Comment by chenilim** [comment]
This should be addressed with the permission and nav changes, though I believe favorites may come a bit later, so keeping this open for now. cc @asaadmahmood, @wuwinson.

**Comment by wuwinson** [comment]
That's correct @chenilim. The new permission and nav changes will allow users to assign boards to categories. Favorites will be coming out a bit later, but we'll support custom categories in Phase 1, so the workaround could be naming a custom category as "Favorites" or something else that appears on top of the LHS. See below for reference:

![148150196-acef17de-02d4-419b-ba96-4ade3ff3089d](https://user-images.githubusercontent.com/93531870/148151118-f7b1509c-9f2b-40b9-b964-4de3509ac222.png)
